client/state/plugins/installed/selectors.js

```diff
@@ -134,6 +137,39 @@ export const getPlugins = createSelector(
 	}
 );
 
+export const getAllDotOrgAndDotComPlugins = createSelector(
```

Comment:
Is this change used in this PR?

Reply:
Oh, that is from some performance testing from another PR, well spotted! I am removing it from this PR and the change in `client/my-sites/plugins/plugins-list/plugins-list-dataviews.tsx`